java/vector/src/main/codegen/templates/NullableValueVectors.java

```diff
@@ -699,6 +699,14 @@ public void reset(){
       setCount = 0;
       <#if type.major = "VarLen">lastSet = -1;</#if>
     }
+
+    public void setLastSet(int value) {
+      <#if type.major = "VarLen">lastSet = value;</#if>
```

Comment:
Writing the unit tests. 

Yes, the ListVector also has lastSet. Will add the API for ListVector as well and unit tests for each vector type; NullableVarCharVector, NullableVarBinaryVector, ListVector.